orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -332,13 +342,18 @@ def _initialize_module_gradient_graph_builder(self):
         # TODO: PyTorch exporter bug: changes the initializer order in ONNX model
         initializer_names = [p[0]
                              for p in self._flattened_output_module.named_parameters()]
+        initializer_names_to_train = []
+        if torch.is_grad_enabled():
+            initializer_names_to_train = [p[0]
+                for p in self._flattened_output_module.named_parameters() if p[1].requires_grad]
         onnx_initializer_names = {
             p.name for p in self._onnx_inference.graph.initializer}
-        initializer_names = [
-            p for p in initializer_names if p in onnx_initializer_names]
+        initializer_names_to_train = [
+            p for p in initializer_names_to_train if p in onnx_initializer_names]
 
         # Build full training graph
         grad_builder_config = C.ModuleGradientGraphBuilderConfiguration()
+        grad_builder_config.initializer_names = initializer_names
         grad_builder_config.initializer_names_to_train = initializer_names
```

Comment:
Should this be set to `initializer_names_to_train`? (Commenting on the before copy, since the line wasn't part of the diff and so GitHub won't let me comment on this line in the new version :).)

Reply:
yes, this was the result of the merging the dev on master and me resolving my conflicts. Will fix. Thanks.